.husky/set-node.sh

```
@@ -135,9 +134,10 @@ install_package_manager() {
 
 # Node.js 설치/업데이트
 install_or_update_node() {
-  local os=$1
-  local version=$2
-  local package_manager=$(get_package_manager)
+  local os="$1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)